tools/cellxgene_census_builder/src/cellxgene_census_builder/__main__.py

```diff
@@ -150,11 +155,30 @@ def do_prebuild_checks(args: CensusBuildArgs) -> bool:
     return True
 
 
+def do_log_environment(args: CensusBuildArgs) -> bool:
+    # Log pip freeze output
+    import pkg_resources
+
+    deps = ""
+    for dep in pkg_resources.working_set:
+        deps += f"{dep.as_requirement()}\n"
+    logging.info(f"Pip freeze:\n{deps}")
+
+    # Log TileDB Config
+    logging.info(f"TileDB Config:\n{DEFAULT_TILEDB_CONFIG}")
```

Comment:
This looks hacky, but the consolidate step is referencing this global variable, so I think this is the only way to do this.

Reply:
I don't think this is useful information, as it is just printing out the hard-coded constant.

What we want to run (i.e., what John requested) is: `tiledbsoma.show_package_versions()`

Or if from entrypoint.sh, `python -c 'print(tiledbsoma.show_package_versions())'`

Reply:
In the other logging PR, `DEFAULT_TILEDB_CONFIG` will be changed to add logging config, so not entirely hard-coded anymore. Which led me to this thought: https://github.com/chanzuckerberg/cellxgene-census/pull/767#issuecomment-1719910261